sqlite3.go

```diff
@@ -1813,6 +1858,24 @@ func (c *SQLiteConn) SetLimit(id int, newVal int) int {
 	return int(C._sqlite3_limit(c.db, C.int(id), C.int(newVal)))
 }
 
+// FileControl invokes the xFileControl method on a given database. The dbName
+// is the name of the database. It will default to "main" if left blank. The op
+// is one of the opcodes prefixed by "SQLITE_FCNTL_". The arg argument and
+// return code are both opcode-specific. Please see the SQLite documentation.
+//
+// See: sqlite3_file_control, https://www.sqlite.org/c3ref/file_control.html
+func (c *SQLiteConn) FileControlInt(dbName string, op int, arg int) int {
```

Comment:
I think we should name this `SetFileControlInt` or similar, since you are only allowing for them to set the value. (For example, to get the current persistent WAL mode, you need to pass a pointer to -1, and then SQLite will change it to point to the current mode, but as written it won't actually return this value.)
Also, it should return `error`, not `int`. See `lastError`. On that note, you should probably document that this isn't thread-safe, or you need to take the lock to make it so. (It's really `c.lastError` that is not thread-safe, in the sense that that if some other operation happens first, it will yield the wrong result.)

Reply:
@rittneje Thanks for the feedback. I updated the method name, changed the return type to `error`, and noted in the docs that the method is not thread-safe.